src/views/AboutProjectView.vue

```diff
@@ -13,7 +13,7 @@ export default {
 </script>
 <template>
   <div class="container px-5 py-3" style="min-height: 60vh">
-    <h2 class="pb-3">The Project</h2>
+    <h2 class="pb-3">The G2P Project</h2>
```

Comment:
The dropdown should also be updated to `The G2P Project`

Reply:
changed in latest commit